contract/test/orca-contract.test.js

```diff
@@ -135,6 +140,30 @@ const makeTestContext = async t => {
           client_id: 'client-0',
           connectionDetails: `${name} connection details`,
         });
+      } else if (key === 'vbankAsset') {
+        return harden({
+          blockHeight: '1',
+          values: () => [
+            {
+              body: [
+                [
+                  'ubld',
+                  {
+                    brand: '$0.Alleged: BLD brand',
```

Comment:
The value here should be a brand, not a string, right?

Why use a custom namehub implementation? `boot-tools.js` sets up `agoricNames` and `agoricNamesAdmin` in the boostratp promise space.

Reply:
seeing this when I try to using agoricNames from boot-tools as you mention here:

```console
orca-contract › orchestrate - osmosis makeAccount and fund returns a ContinuingOfferResult
  Rejected promise returned by test. Reason:

  Error {
    message: 'chain not found:agoric',
  }

  › makeError (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/ses/src/error/assert.js:349:61)
  › eval (.../async-flow/src/convert.js:100:1)
  › innerConvert (.../async-flow/src/convert.js:57:8)
  › convertRecur (.../async-flow/src/convert.js:26:8)
  › Alleged: replayMembrane.convert [as guestToHost] (.../async-flow/src/convert.js:76:1)
  › eval (.../async-flow/src/async-flow.js:265:37)
```
and
```console
Error#1: "nameKey" not found: "chain"
    at makeError (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/ses/src/error/assert.js:349:61)
    at Function.fail (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/ses/src/error/assert.js:481:20)
    at NonNullish (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@agoric/internal/src/errors.js:14:10)
    at Object.lookup (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@agoric/vats/src/nameHub.js:177:15)
    at In "lookup" method of (NameHubKit nameHub) [as lookup] (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@endo/exo/src/exo-tools.js:171:24)
    at localApplyMethod (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@endo/eventual-send/src/local.js:126:18)
    at Object.applyMethod (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@endo/eventual-send/src/handled-promise.js:455:16)
    at dispatchToHandler (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@endo/eventual-send/src/handled-promise.js:156:22)
    at doDispatch (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@endo/eventual-send/src/handled-promise.js:486:7)
    at win (file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@endo/eventual-send/src/handled-promise.js:505:26)
    at file:///Users/jovonni/Documents/projects/dapp-orchestration-basics/node_modules/@endo/eventual-send/src/handled-promise.js:523:20
```

digging